test/node/stat/corcov.ts

```diff
@@ -0,0 +1,41 @@
+import { correlationMatrix, covarianceMatrix } from '../../../src/stat/corcov';
+import '@tensorflow/tfjs-backend-cpu';
```

Comment:
We could use `mocha --require tests/setup.js` to avoid adding this line every time. See https://mochajs.org/#command-line-usage.